.github/CODEOWNERS

```diff
@@ -1,9 +1,7 @@
 # configuration for authorized and required reviewers
-* @DANS-KNAW/easy-leads
 
 # code
 # no overrides
 
 # policies
-/.github/PULL_REQUEST_TEMPLATE.md @DANS-KNAW/easy-leads
-/.github/CODEOWNERS @DANS-KNAW/easy-leads
+
```

Comment:
I suppose you should have replaced with @DANS-KNAW/dataversedans

The archetype also missed some lines for [PULL_REQUEST_TEMPLATE](https://github.com/DANS-KNAW/dd-easy-worflows-poc/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
It should trigger the review process with notifications and get it on an overview like for [EASY](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+team%3ADANS-KNAW%2Feasy)



Its content should become something like
```
Fixes EASY-

* 
* 
* 

#### Where should the reviewer @DANS-KNAW/dataversedans start?

#### How should this be manually tested?

#### Related pull requests on github

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..

```